### ISSUE TITLE
Test::PerlTidy fails its tests when run with the latest Perl::Tidy

### ISSUE THREAD

**Issue by dakkar** [shlomif/Test-PerlTidy#7]
I suspect some defaults have changed upstream, and the tests need to be updated.

Probably related to https://github.com/perltidy/perltidy/issues/93

[fail.log](https://github.com/shlomif/Test-PerlTidy/files/8920297/fail.log)

**Comment by eserte** [comment]
Statistical analysis suggests that this happens if Perl::Tidy 20220613 is installed (negative theta means "bad"):
```
****************************************************************
Regression 'mod:Perl::Tidy'
****************************************************************
Name           	       Theta	      StdErr	 T-stat
[0='const']    	      1.0000	      0.0000	20916417934311208.00
[1='eq_20210111']	      0.0000	      0.0000	   3.79
[2='eq_20210402']	      0.0000	      0.0000	   8.38
[3='eq_20210625']	      0.0000	      0.0000	  10.10
[4='eq_20210717']	      0.0000	      0.0000	   8.31
[5='eq_20211029']	      0.0000	      0.0000	   9.48
[6='eq_20220217']	      0.0000	      0.0000	   8.42
[7='eq_20220613']	     -1.0000	      0.0000	-19093989874713092.00

R^2= 1.000, N= 177, K= 8
****************************************************************
```

**Comment by dakkar** [comment]
yes, that's what I meant by "latest", https://github.com/perltidy/perltidy/blob/master/CHANGES.md#2022-06-13 confirms it:

> Added vertical alignment for qw quotes and empty parens in 'use'
> statements (see issue #git 93).  This new alignment is 'on' by default

**Comment by Tux** [comment]
This "breakage" now will cause the package `perl-Test-PerlTidy` to be removed from the openSUSE Tumbleweed distribution if not fixed shortly. Just noting. I have no influence there (other than being able to tell them that someone is working on it if that were the case)

**Comment by shlomif** [comment]
seems fixed in 20220922. Thanks.
